README.md

```diff
@@ -84,6 +84,21 @@ This does three things:
 
 You can then set Prettier's own options inside a `.prettierrc` file.
 
+3. In order to support special ESLint plugins (e.g. [eslint-plugin-react](https://github.com/yannickcr/eslint-plugin-react)), add extra exclusions for the plugins you use like so:
```

Comment:
I think the numbering here is a bit confusing because there is a separate numbered list directly above this line. Can you change the list above this one to use bullet points (by prefixing each line with `*` rather than numbers?

Reply:
Sure, I’ve changed it.